tests/Query/CombinedFieldsQueryTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Elastica\Test\Query;
+
+use Elastica\Document;
+use Elastica\Index;
+use Elastica\Mapping;
+use Elastica\Query;
+use Elastica\Query\CombinedFields;
+use Elastica\ResultSet;
+use Elastica\Test\Base as BaseTest;
+
+/**
+ * @internal
+ */
+class CombinedFieldsQueryTest extends BaseTest
+{
+    private static $data = [
+        ['id' => 1, 'title' => 'Rodolfo', 'body' => 'Moraes',   'abstract' => 'Lorem'],
+        ['id' => 2, 'title' => 'Tristan', 'body' => 'Maindron', 'abstract' => 'Dolor'],
+        ['id' => 3, 'title' => 'Monique', 'body' => 'Maindron', 'abstract' => 'Ipsum'],
+        ['id' => 4, 'title' => 'John',    'body' => 'not Doe',  'abstract' => 'Consectetur'],
+    ];
+
+    /**
+     * @group functional
+     */
+    public function testMinimumShouldMatch(): void
+    {
+        // Note that the OR operator is the default.
+        $combinedFields = new CombinedFields();
+        $combinedFields->setQuery('Tristan Maindron');
+        $combinedFields->setFields(['title', 'body', 'abstract']);
+        $combinedFields->setMinimumShouldMatch('2<100%');
```

Comment:
Interesting one, was not aware of this syntax. Something learned today: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-minimum-should-match.html